network-api/networkapi/wagtailpages/templates/wagtailpages/campaign_page.html

```diff
@@ -11,6 +11,13 @@
       {% endif %}"
     >
 
+      <h1 class="h1-heading {% if uses_menu %}hidden-sm-down{% endif %}">
```

Comment:
this is the exact same code as in mini_site_name_space.html, can we turn this into a fragment and `{% include "h1-heading.html" %}` that in both places?